src/builder/javascript.js

```diff
@@ -1,16 +1,29 @@
 import { transformFile } from 'babel-core'
 import { default as deferred } from 'thenify'
 import { default as compat } from 'babel-plugin-add-module-exports'
-import { default as es2015 } from 'babel-preset-es2015'
-import { default as umd } from 'babel-plugin-transform-es2015-modules-umd'
+import { default as ecmascript } from 'babel-preset-latest'
+
+const babelrc = process.env.NODE_ENV !== 'test'
 
 export default function configure(pkg, opts) {
   return async function process(name, file) {
-    let presets = [es2015]
-      , plugins = [umd]
+    const { es2017
+          , modules
+          , ['add-module-exports']: addModuleExports
+          } = opts.flags
+
+    let presets = [
```

Comment:
Again, excellent commentary – much obliged! :o)

I don't necessarily agree or disagree, I'm very much on the fence. I started #36 as a place for us to continue this discussion. Or do you think this is make-or-break for merging this PR and we really should hash this out first?